services/dashboard/views/apps/streamingapp/streamingapp.html

```diff
@@ -8,13 +8,13 @@
       </h4>
       <h5 ng-bind="app.description"></h5>
     </div>
-    <div class="col-md-2 col-sm-3 col-xs-4">
+    <div class="col-md-2 col-sm-3 col-xs-3">
```

Comment:
I didn't find there is any alignment issue. Here is the screen shot for iPhone 4.
![align](https://cloud.githubusercontent.com/assets/2595532/12382031/5c70c6a6-bdd0-11e5-86a3-2e19ec0f51d5.jpg)